manticore/core/smtlib/visitors.py

```diff
@@ -147,6 +147,7 @@ def visit_Operation(self, expression, *operands):
         else:
             self._print('...')
         self.indent -= 2
+        return ''
```

Comment:
The problem comes from `Visitor._method`, which tries to iterate on all the possible `visit_` methods until one does not return `Ǹone`, that's why I simply added a `return ''` (the return value is never used in fact). If it does make more sense it can return the string representation instead. Another solution could be to write a `_method` for `PettyPrinter` which stops to iterate if a corresponding visitor is found.

For the call to `visit` in `visit_operation`, I guess that the recursive approach was chosen for the PrettyPrinter (instead of using a stack), as it makes easier to keep the consistency of the indentation (but it's just a guess)